operations/configuration_management/Management_Node_Image_Customization.md

```diff
@@ -17,29 +18,98 @@ with minor command modifications.
+      by examining the boot parameters used to boot the NCN in question.
+
+      1. Set the `BOOTED_NCN` variable to the hostname of the NCN that is booted using the image that is to be modified.
+
+         > For example, `ncn-w001` or `ncn-s002`.
+
+         ```bash
+         BOOTED_NCN=ncn-<msw###>
+         ```
+
+      1. Extract the S3 path prefix for the image used to boot the chosen NCN.
+
+         This prefix corresponds to the IMS image ID of the boot image.
+
+         ```bash
+         NCN_IMS_IMAGE_ID=$(ssh "${BOOTED_NCN}" sed \
+                                  "'s#\(^.*[[:space:]]\|^\)metal[.]server=[^[:space:]]*/boot-images/\([^[:space:]]\+\)/rootfs.*#\2#'" \
+                                  /proc/cmdline)
+         echo "${NCN_IMS_IMAGE_ID}"
+         ```
+
+         The output should be a UUID string. For example, `8f41cc54-82f8-436c-905f-869f216ce487`.
+
+         > The command used in this substep is extracting the location of the NCN image from the `metal.server` boot parameter in
+         > the `/proc/cmdline` file on the booted NCN. For more information on that parameter, see
+         > [`metal.server` boot parameter](../../background/ncn_kernel.md#metalserver).
+
+    * If not doing a CSM upgrade, and if the image to be modified is the boot image of an NCN that is not currently booted, then
+      identify the image by examining that NCN's boot parameters in the
+      [Boot Script Service (BSS)](../../glossary.md#boot-script-service-bss).
+
+      > This procedure can also be used for an NCN which is booted, but the procedure obtains the boot image which will be used the next time
+      > that the NCN boots. This may not necessarily match what the NCN is currently booted with.
+
+      1. Set `NCN_XNAME` to the [component name (xname)](../../glossary.md#xname) of the NCN whose boot image is to be used.
+
+         ```bash
+         NCN_XNAME=<xname>
+         ```
+
+      1. Extract the S3 path prefix for the image that will be used on the next boot of the chosen NCN.
+
+         ```bash
+         NCN_IMS_IMAGE_ID=$(cray bss bootparameters list --name "${NCN_XNAME}" --format json | \
+                              jq -r '.[0].params' | \
+                              sed 's#\(^.*[[:space:]]\|^\)metal[.]server=[^[:space:]]*/boot-images/\([^[:space:]]\+\)/rootfs.*#\2#')
+         echo "${NCN_IMS_IMAGE_ID}"
+         ```
+
+         The output should be a UUID string. For example, `8f41cc54-82f8-436c-905f-869f216ce487`.
+
+         > The command used in this substep is extracting the location of the NCN image from the `metal.server` boot parameter for the
+         > NCN in BSS. For more information on that parameter, see [`metal.server` boot parameter](../../background/ncn_kernel.md#metalserver).
+
+1. (`ncn-mw#`) Obtain the NCN image's associated artifacts (SquashFS, kernel, and `initrd`).
```

Comment:
I don't understand why this step is added here. These files are getting downloaded and then not used anywhere else in this documentation, as far as I can tell.

This looks like part of the old process that was used to download the image artifacts from S3 in order to then create an IMS image record, re-upload the image artifacts to the appropriate path in IMS, create the IMS image manifest that references those artifacts, and then linking that manifest to the IMS image record. In CSM 1.4, this is already done for us as part of the CSM installation.